docs/lang/articles/basic/operator.md

```diff
@@ -83,6 +83,15 @@ default numerical types.
 | `a << b`                | left-shift `a` by `b` bits          |
 | `a >> b`                | right-shift `a` by `b` bits         |
 
+:::note
+
+The `>>` operation denotes the
+[Shift Logical](https://en.wikipedia.org/wiki/Logical_shift) Right (SHR) operation. For the
+[Shift Arithmetic](https://en.wikipedia.org/wiki/Arithmetic_shift) Right (SAR) operation,
+consider using `ti.bit_shr()` or `ti.bit_shl()` for the left shift operation.
```

Comment:
sal and shl are synonyms so we can just use `<<`

Reply:
ok, let's mention both for completeness.